processing/data_collection/gazette/settings.py

```diff
@@ -8,4 +8,4 @@
     "gazette.pipelines.ExtractTextPipeline": 200,
 }
 FILES_STORE = "/mnt/data/"
-QUERIDODIARIO_EXTRACT_TEXT_FROM_FILE = True
\ No newline at end of file
+QUERIDODIARIO_EXTRACT_TEXT_FROM_FILE = True
```

Comment:
this same change is in another of your PRs, it will cause conflicts once that one is merged

Reply:
We decided to rebase this once the first one gets merged.